trusted-types/trusted-types-source-file-path.html

```diff
@@ -50,9 +44,9 @@
   let script_src = script_origin +
```

Comment:
I know this isn't directly related to this PR but the way this URL is constructed doesn't work if you access WPT through https


  `let script_src = new URL("/trusted-types/support/set-inner-html.js", location.href).href;` This would be better I think

Reply:
Hmm but that might be wrong for the below tests? Maybe leave it as is for now but with a note explaining this mgiht not be ideal?

Reply:
OK, I guess we should try to fix that, but this PR is already big and we are not going to fix all existing issues in these tests. I'll try to fix that in a follow-up PR.

Reply:
checking https://github.com/web-platform-tests/wpt/blob/master/common/get-host-info.sub.js it seems we want ORIGIN and REMOTE_ORIGIN instead, but I'm not really sure so let's handle this separately